public/src/modules/navigator.js

```diff
@@ -165,7 +166,9 @@ define('navigator', ['forum/pagination', 'components'], function (pagination, co
 		index = index > count ? count : index;
 
 		$('.pagination-block .pagination-text').translateHtml('[[global:pagination.out_of, ' + index + ', ' + count + ']]');
-		$('.pagination-block .progress-bar').width((index / count * 100) + '%');
+		var fraction = $(window).scrollTop() / ($(document).height() - $(window).height());
+		$('.pagination-block meter').val(fraction);
```

Comment:
@julianlam This code makes the HTML5 meter work, my initial thought was to replace the "old" bootstrap progress bar, but this would introduce some breaking changes.

Reply:
@julianlam Using the HTML5 tag, seems to make the bar set it's value more instantly...
At least in my perception the div + progress bar is lagging a couple fractions of a second behind, which is not or not that much the case using the meter tag.
Since removing the current behaviour would introduce a breaking change, I thought it would be a good idea to allow for both, so that template designers could use the "better solution" when updating their templates.
Removing the current functionality using the bootstrap bar would be a task for a major update

Reply:
> At least in my perception the div + progress bar is lagging a couple fractions of a second behind, which is not or not that much the case using the meter tag.

This "lag" would be caused by a transition set for the div in its CSS. Removing the transition (or lowering its duration) should make it update at the same speed as the meter tag.